vcd/resource_vcd_vapp_network.go

```diff
@@ -1,20 +1,25 @@
 package vcd
 
 import (
+	"bytes"
 	"fmt"
-	"log"
-
+	"github.com/hashicorp/terraform-plugin-sdk/helper/hashcode"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/vmware/go-vcloud-director/v2/govcd"
 	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"log"
+	"strings"
 )
 
 func resourceVcdVappNetwork() *schema.Resource {
```

Comment:
Part of the fields here and in data sources are missing "Descriptions" and I think this could be a good tidy up opportunity.

Reply:
Add descriptions everythere